src/controls.lua

```diff
@@ -1,48 +1,48 @@
-local Controls = {}
-
 local Log = require("log")
 
 local log = Log({on = true})
 local mouse = love.mouse
 local keyboard = love.keyboard
 
-local bindingsFile = "config/bindings.lua"
+local Controls = class()
 
--- TODO: We probably want to organize this file as a class with methods?
+local bindingsFile = "config/bindings.lua"
 
-function Controls.getOrders(controls)
+function Controls:getOrders()
 	local orders = {}
 	for key, order in pairs(Controls.shipCommands) do
-		if Controls.isDown(controls[key]) then
+		if Controls.isDown(self.bindings[key]) then
 			table.insert(orders, order)
 		end
 	end
 	return orders
 end
 
-function Controls.test(mode, controls, source, button)
+function Controls:test(mode, source, button)
 	for key, value in pairs(Controls[mode]) do
-		local control = controls[key]
+		local control = self.bindings[key]
 		if control[1] == source and control[2] == button then
 			return value
 		end
 	end
 end
 
-function Controls.setCursor(control, Cursor)
-	local cursorChange
-	if control[2] == "xAxis" then
-		cursorChange = control[1].getX()
-	elseif control[2] == "yAxis" then
-		cursorChange = control[1].getY()
+function Controls:getCursorPosition(oldX, oldY)
+	local cursor = self.bindings.cursor
+	local device = cursor.device
+	local xChange, yChange
+
+	if device == mouse then
+		xChange, yChange = device.getPosition()
 	else
-		cursorChange = control[1]:getAxis(control[2])
+		xChange = device:getAxis(cursor.x)
+		xChange = device:getAxis(cursor.y)
 	end
```

Comment:
I rewrote setCursor to 1: use `self` instead of a user-provided argument, and 2: return both X and Y at once.